# Disables automatic pretrained weight downloading; weights are loaded separately.
#
diff --git a/modeling/backbone/mobilenet.py b/modeling/backbone/mobilenet.py
index 6fff541..0a1abf2 100644
--- a/modeling/backbone/mobilenet.py
+++ b/modeling/backbone/mobilenet.py
@@ -121,14 +121,7 @@ class MobileNetV2(nn.Module):
         return x, low_level_feat

     def _load_pretrained_model(self):
-        pretrain_dict = model_zoo.load_url('http://jeff95.me/models/mobilenet_v2-6a65762b.pth')
-        model_dict = {}
-        state_dict = self.state_dict()
-        for k, v in pretrain_dict.items():
-            if k in state_dict:
-                model_dict[k] = v
-        state_dict.update(model_dict)
-        self.load_state_dict(state_dict)
+        pass

     def _initialize_weights(self):
         for m in self.modules():
